backend/backend.go

```diff
@@ -0,0 +1,45 @@
+package backend
+
+import (
+	"context"
+
+	"github.com/skroutz/downloader/job"
+)
+
+// Known implementations
```

Comment:
These constants should be part of the notifier package instead, since it's skroutz-specific logic. They shouldn't pollute the backend package, which is backend-agnostic.